includes/fbs.availability.inc

```diff
@@ -80,6 +80,7 @@ function fbs_availability_holdings($provider_ids) {
       // And FBS can't tell us this.
       'reserved_count' => 0,
       'is_periodical' => FALSE,
+      'reservation_count' => $item->reservations
```

Comment:
reservation_count is used here: https://github.com/artesis/fbs/blob/master/includes/fbs.availability.inc#L214